README.md

```diff
@@ -0,0 +1,19 @@
+# Mossley Heritage Society's website
+
+This site is build with jekyll and can be viewed at https://mossleyheritage.org/
```

Comment:
maybe a short requirements section covering ruby version, rbenv, ruby-gemset and ruby-build?

Reply:
would you be up for writing a suggestion for this, I think you know more than me about it.